src/com/sun/ts/tests/jaxws/api/javax_xml_ws/WebServiceContext/WSWebServiceContext_wsappclient_vehicle_client.jar.sun-application-client.xml

```diff
@@ -28,11 +28,11 @@
         <localpart>HelloPort</localpart>
       </wsdl-port>
       <stub-property>
-        <name>javax.xml.ws.security.auth.password</name>
+        <name>jakarta.xml.ws.security.auth.password</name>
```

Comment:
Can you update the copyright year in the xml files, too?

Reply:
Done.